docs/source/tts/checkpoints.rst

```diff
@@ -156,6 +156,6 @@ End2End models
 Codec models
 ^^^^^^^^^^^^
 .. csv-table::
-   :file: data/models_codec.csv
+   :file: data/ngc_models_codec.csv
    :align: left
```

Comment:
Also curious if this is intended ? Just making sure

Reply:
Pulled the @hemildesai 's PR and I guess this is coming from his PR.